src/Text.Analyzers/UnitTests/IdentifiersShouldBeSpelledCorrectlyTests.cs

```diff
@@ -1,8 +1,473 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Testing;
+using Test.Utilities;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Text.Analyzers.IdentifiersShouldBeSpelledCorrectlyAnalyzer,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
 
 namespace Text.Analyzers.UnitTests
 {
     public class IdentifiersShouldBeSpelledCorrectlyTests
     {
+        public enum DictionaryType
+        {
+            Xml,
+            Dic,
+        }
+
+        public static IEnumerable<object[]> MisspelledMembers
+            => new[]
+            {
+                new object[] { CreateTypeWithConstructor("<?>Clazz", isStatic: false), "Clazz", "Clazz.Clazz()" },
```

Comment:
😕 I don't understand the use of `<?>`. Maybe ported Java code?

Reply:
It was my version of markupKey